start.sh

```diff
@@ -43,6 +44,17 @@ https://bw.{\$NC_DOMAIN}:443 {
     # import GEOFILTER
 CADDY
 
+		if [ -f /nextcloud/admin/files/nextcloud-aio-caddy/allowedIps-bw ]; then 
+		allowedIPs=$(cat /nextcloud/admin/files/nextcloud-aio-caddy/allowedIps-bw)
+		
+	    cat << CADDY >> /Caddyfile
+	@public_networks not remote_ip $allowedIPs
+        respond @public_networks 403 {
+			close
+		}
+CADDY
+	fi
```

Comment:
We need to name the file a bit differently and at least add a check that the string is not empty for all of the services. Here an example:
```suggestion
		if [ -f /nextcloud/admin/files/nextcloud-aio-caddy/allowed-IPs-bitwarden.txt ]; then 
		allowedIPs=$(cat /nextcloud/admin/files/nextcloud-aio-caddy/allowed-IPs-bitwarden.txt)
		if [ -n "$allowedIPs" ]; then
	        cat << CADDY >> /Caddyfile
	@public_networks not remote_ip $allowedIPs
        respond @public_networks 403 {
			close
		}
CADDY
	    fi
    fi
```